tst/teststandard/ctblmono.tst

```diff
@@ -0,0 +1,31 @@
+gap> START_TEST("ctblmono.tst");
+
+#
+# the following test comes from https://github.com/gap-system/gap/issues/4452
+# this used to give method not found for non-solvable groups
+#
+gap> IsMinimalNonmonomial(SL(2,3));
+true
+gap> IsMinimalNonmonomial(GL(2,3));
+false
+gap> IsMinimalNonmonomial(PSL(2,4));
+true
+gap> IsMinimalNonmonomial(PGL(2,5));
+false
+gap> IsMinimalNonmonomial(PSL(2,7));
+false
```

Comment:
That's actually wrong (which agrees with Thompson's result and the other changes I suggested above. Indeed:
```
gap> all:=AllSubgroups(PSL(2,7));; Length(all); Number(all, IsMonomialGroup);
179
178
```

So we need this:
```suggestion
gap> IsMinimalNonmonomial(PSL(2,7));
true
```